PinkPong/Assets/Scripts/PaddleMovement.cs

```diff
@@ -0,0 +1,42 @@
+using UnityEngine;
+
+public class PaddleMovement : MonoBehaviour
+{
+    [SerializeField] private Rigidbody2D rb2d;
+    [SerializeField] private PaddleInput input;
+    [SerializeField] private float speed;
+    [SerializeField] private float dashSpeed;
+    [SerializeField] private Transform target;
+
+    private Vector3 startPosition;
+
+    private void Start()
+    {
+        startPosition = transform.position;
+    }
+
+    private void OnEnable()
+    {
+        input.IsDoublePressed += Dash;
+    }
+
+    private void OnDisable()
+    {
+        input.IsDoublePressed -= Dash;
+    }
+
+    public void ResetPaddle()
+    {
+        transform.position = startPosition;
+    }
+
+    public void Move(float movement)
```

Comment:
```suggestion
    public void Move(RotationDirection direction)
```

where `RotationDirection` is 
```c#
enum RotationDirection {
    Clockwise,
    CounterClockWise
}
```